tests/utils/exec/rosacli/autoscaler_service.go

```diff
@@ -12,10 +13,10 @@ import (
 type AutoScalerService interface {
 	ResourcesCleaner
 
-	CreateAutoScaler(clusterID string, flags ...string) (bytes.Buffer, error)
```

Comment:
Instead of changing this I would send an empty string to the `CreateAutoScaler` method and then into the method, if empty clusterID, then don't send any in flags.

Reply:
Sorry I didn't understand you.